### DISCUSSION TITLE
What comes next in this sequence?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#897] [Galaxy Trivia] [answered]
```
T  A  O  T  B  I  E  S  O  T  R  H  N  E  O  Q  T  U  T  E  O  S  B  T  E  I  T  O  H  ?
```

**Comment by peterbaccaro** [comment, marked as answer]
Taking all the odd letters:

**T ~~A~~ O ~~T~~ B ~~I~~ E ~~S~~ O ~~T~~ R ~~H~~ N ~~E~~ O ~~Q~~ T ~~U~~ T ~~E~~ O ~~S~~ B ~~T~~ E ~~I~~ T ~~O~~ H**

You get: 

**TO BE OR NOT TO BE TH**

Taking all the even letters:

**~~T~~ A ~~O~~ T ~~B~~ I ~~E~~ S ~~O~~ T ~~R~~ H ~~N~~ E ~~O~~ Q ~~T~~ U ~~T~~ E ~~O~~ S ~~B~~ T ~~E~~ I ~~T~~ O ~~H~~**

You get: 

**AT IS THE QUESTIO**

So the next letter in the sequence is **N**.
